Jenkinsfile

```diff
@@ -585,11 +585,11 @@ pipeline {
               git(credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba',
                   url: "git@github.com:elastic/${OPBEANS_REPO}.git",
                   branch: 'main')
-              sh script: ".ci/bump-version.sh ${env.BRANCH_NAME}", label: 'Bump version'
+              sh script: ".ci/bump-version.sh ${env.BRANCH_NAME.replaceAll('^v', '')}", label: 'Bump version'
```

Comment:
https://github.com/elastic/opbeans-dotnet/pull/67 is also another way to do the same, so I decided to use this approach since it's the one calling the script